pages/your_problem/data_analysis.md

```diff
@@ -1,39 +1,65 @@
 ---
 title: Data Analysis
 keywords: optional
-contributors: []
-tags: []
+contributors: [Olivier Collin]
+tags: [Data analysis]
```

Comment:
```suggestion
tags: [analyse]
```
In order for this page to show up in the Data life Cycle page Analysing, it need to be tagged with analyse.

Reply:
Good idea ! 
I completely forgot the link with the Data life Cycle....